### DISCUSSION TITLE
Work without Fooocus running?

### DISCUSSION THREAD

**Discussion by ILikeToasters** [toutjavascript/Fooocus-Log-Viewer#23] [Ideas]
Was wondering how difficult it would be to have it so the log viewer could be used when Fooocus is not running. I know some people use online services to run Fooocus and then save their images locally for example. Also it would be useful to not have to start Fooocus if just looking for an image. I have not had a chance to dig into the code myself plus I am not an expert on the subject.

**Comment by toutjavascript** [comment]
Hello
At the moment, Fooocus is used as a web server to display the result of log file scan
A web server is necessary. It could be another one but it is simpler with Fooocus
But I understand the need of a distant web service

Thanks
